src/components/DoctorsPage/DoctorsCard/Schedule/Schedule.tsx

```diff
@@ -83,13 +98,23 @@ const Schedule: React.FC<IScheduleProps> = ({ workTimeData }) => {
       <ItemWrapper>{renderItem()}</ItemWrapper>
       {stateSchedule.viewSuccessMessage && (
         <SuccessMessage>
-          You have booked a ticket for {stateSchedule.choiceWorkTime.time} on{' '}
-          {stateSchedule.choiceWorkTime.date}. Be healthy!
+          {error ? (
+            <Error errorMessage={error} />
+          ) : (
+            `You have booked a ticket for ${stateSchedule.choiceWorkTime.time} on
+            ${stateSchedule.choiceWorkTime.date}. Be healthy!`
+          )}
         </SuccessMessage>
       )}
-      <Button onClick={() => sendWorkTime()} round variant="contained">
-        Sign Up
-      </Button>
+      {disabledButton ? (
```

Comment:
what for ternary here if buttons are totally the same and differs only with one prop?

Reply:
I don't quite understand how it should be